### DISCUSSION TITLE
How to skip SSL certificate checking?

### DISCUSSION THREAD

**Discussion by grzegorzblaszczyk** [sindresorhus/ky#277] [Q&A] [answered]
When I do this: 

```typescript
import ky from 'https://unpkg.com/ky@0.23.0/index.js';

(async () => {
  class HTTPError extends Error {}
  const response = await ky('https://some-url-with-invalid-ssl-cert.com/file.txt', {
    method: 'GET'
  });

  if (!response.ok) {
    console.log (`Response: ${response}`)
  }

})();
```

I get: 

```
WARN RS - rustls::session:718 - Sending fatal alert BadCertificate
```

How to skip SSL certificate checking in ky?

**Comment by sindresorhus** [comment]
`ky` is using the browser Fetch API underneath. I don't think this is possible.

https://www.google.com/search?q=fetch+skip+ssl+checking&oq=fetch+skip+ssl+checking&aqs=chrome..69i57j0j69i64.3309j0j1&sourceid=chrome&ie=UTF-8

**Comment by bernardoadc** [comment, marked as answer]
In browsers it's not possible. However, if using it in node (for instance, in automated tests using avajs) it can be done like this:

```
(package.json)
  "ava": {
    "environmentVariables": {
      "NODE_TLS_REJECT_UNAUTHORIZED": "0"
    }
  }
```

Use only in development mode, like in tests.
